package/MDAnalysis/analysis/rms.py

```diff
@@ -484,6 +486,63 @@ def save(self, filename=None):
         return filename
 
 
+def rmsf(atomgroup, start=0, stop=-1, step=1, progout=None):
```

Comment:
**Please package as a class.** Leave out plotting if you don't have the time but it should have the same overall form as other "standard" analysis functions.

Otherwise we'll deprecated the function soon again in favor of a class and that seems silly.